utils/sysstat/Makefile

```diff
@@ -31,8 +31,8 @@ define Package/sysstat
   SECTION:=utils
   CATEGORY:=Utilities
   TITLE:=Sysstat performance monitoring tools
-  URL:=http://pagesperso-orange.fr/sebastien.godard/index.html
-  DEPENDS:=$(INTL_DEPENDS)
+  URL:=https://sysstat.github.io/
+  DEPENDS:=$(INTL_DEPENDS) +xz
```

Comment:
Since the 'xz' package is only the host package, could this be 'xz-utils'?

Reply:
It seems that 'xz' can only be specified if the meta package 'xz-utils' is selected. What about 'DEPENDS' in this case?

Reply:
@nxhack you're right, thank you for spotting this! I've pushed the fix in #23514.